### PR TITLE
Added table stories

### PR DESCRIPTION
**Pull request by Lisa159**

### CODE REVIEW

**Review by marcialfps — COMMENTED** [review]
I think it is necessary to add the case where the table has a scrollbar.

**Comment by Lisa159** [comment]
> I think it is necessary to add the case where the table has a scrollbar.

Done

File: lib/src/table/Table.stories.jsx [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+  </>
+);
+
+// const Container = styled.div`
+//   display: flex;
+//   flex-direction: row;
+//   height: 400px;
+//   width: 100%;
+//   margin-bottom: 50px;
+// `;
```
Comment (marcialfps) [inline lib/src/table/Table.stories.jsx:284]: Please delete these comments ✌🏼

**Review by marcialfps — APPROVED** [review]

File: lib/src/table/Table.stories.jsx (hunk 1 above)
Comment (Lisa159) [reply lib/src/table/Table.stories.jsx]: Done

**Pull request merged**